main.go

```diff
@@ -86,40 +98,24 @@ func main() {
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "d27fd235.operator.containers.carbonblack.io",
 		Logger:                 ctrl.Log,
+		Namespace:              operatorNamespace,
```

Comment:
Notice that this means our operator should _not_ be getting any watch updates for resources outside the namespace anymore (regardless of RBAC)

Reply:
The namespace is only applied for namespaced resources in the cache. Non-namespaced resource are always watched.
I tested just in case and the operator re-creates the hooks and priority class if modified 

Comment from the field itself:
```
	// Namespace, if specified, restricts the manager's cache to watch objects in
	// the desired namespace. Defaults to all namespaces.
	//
	// Note: If a namespace is specified, controllers can still Watch for a
	// cluster-scoped resource (e.g Node). For namespaced resources, the cache
	// will only hold objects from the desired namespace.
```